cmd/setup.go

```diff
@@ -20,6 +20,9 @@ var (
 		Short: "setup e2e tests",
 		RunE:  runSetup,
 	}
+)
+
+var (
```

Comment:
I think it makes sense to separate the command and its flags in different blocks.